### ISSUE TITLE
Collection inputs and textarea have no focus styles

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#5574]
Using a keyboard to focus the form fields the inputs and text area have no focus styling.

<img width="502" alt="stuart_s_collection_ _add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/42621296-540bfda0-85b5-11e8-9198-3b102c0606ff.png">

See also https://github.com/mozilla/addons-frontend/issues/5573 in-case something generic can be setup that handles this for all fields.

**Comment by vcarciu** [comment]
Verified as fixed : 
![focusstyles](https://user-images.githubusercontent.com/1583842/42688379-b101b168-86a4-11e8-9798-ad010105e7d7.gif)
